src/main/java/racingcar/util/Censor.java

```diff
@@ -0,0 +1,67 @@
+package racingcar.util;
+
+import static racingcar.util.constant.GameRule.MAX_NICKNAME_LENGTH;
+import static racingcar.util.messageContent.ExceptionMessageContent.NICKNAME_LENGTH_ERROR;
+import static racingcar.util.messageContent.ExceptionMessageContent.NICKNAME_SPACE_ERROR;
+import static racingcar.util.messageContent.ExceptionMessageContent.NICKNAME_UNIQUE_ERROR;
+import static racingcar.util.messageContent.ExceptionMessageContent.NOT_INTEGER_ERROR;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Censor {
+
+    public void checkInputForNickname(String input) {
+        commonValid(input);
+        List<String> playerList = TypeTransducer.strToList(input);
+        checkLength(playerList);
+        checkUniqueValue(playerList);
+    }
+
+    public void checkInputForGameTime(String input) {
+        commonValid(input);
+        checkInteger(input);
+    }
+
+    private void commonValid(String input) {
+        if (input == null) {
+            throw new IllegalArgumentException(NICKNAME_SPACE_ERROR.getContent());
+        }
+
+        if (input.trim().isEmpty()) {
```

Comment:
`trim().isEmpty()` 대신 `isBlank()`를 사용해 보시는 건 어떠실까요?

```suggestion
        if (input.isBlank()) {
```